src/libraries/Microsoft.Extensions.Logging/tests/Common/TraceSourceLoggerTest.cs

```diff
@@ -51,10 +55,37 @@ public static void MultipleLoggers_IsEnabledReturnsCorrectValue(SourceLevels fir
                 .AddTraceSource(secondSwitch));
 
             var logger = factory.CreateLogger("Test");
-
+            
```

Comment:
nit remove whitespace

Reply:
I can remove the whitespaces, but to be clear this is not one of my tests, it was already there before and I didn't check the other tests. Br